incubator/node-red/image/Dockerfile-stack

```diff
@@ -0,0 +1,30 @@
+FROM node:10
+
+ENV APPSODY_MOUNTS=/:/project/user-app
+ENV APPSODY_DEPS=/project/user-app/node_modules
+
+ENV APPSODY_WATCH_DIR=/project/user-app
+ENV APPSODY_WATCH_IGNORE_DIR=/project/user-app/node_modules
+ENV APPSODY_WATCH_REGEX="^.*.json$"
+
+ENV APPSODY_INSTALL="npm install --prefix user-app && npm audit fix --prefix user-app"
+
+ENV APPSODY_RUN="npm start"
+ENV APPSODY_RUN_ON_CHANGE="npm start"
+ENV APPSODY_RUN_KILL=true
+
+ENV APPSODY_DEBUG="npm run debug"
+ENV APPSODY_DEBUG_ON_CHANGE="npm run debug"
+ENV APPSODY_DEBUG_KILL=true
+
+ENV APPSODY_TEST="npm test"
+
+COPY ./project /project
```

Comment:
You also need to copy the `/config` folder into the container so `appsody deploy` has access to the deployment config

Reply:
Done